### DISCUSSION TITLE
Possible bug : Oink Code / Registration Code cannot start with '0'

### DISCUSSION THREAD

**Discussion by DoughnutPeach** [Security-Onion-Solutions/securityonion#12473] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

cloud

### Hardware Specs

Exceeds minimum requirements

### CPU

32

### RAM

16

### Storage for /

500

### Storage for /nsm

500

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

My Oink Code starts with a '0'.

I followed the documentation and pasted the code into SOC > Administration > idstools > config > Registration Code.

However, the value stored in `/opt/so/saltstack/local/pillar/idstools/soc_idstools.sls` is missing the starting '0' character.

I had to manually edit this file for `so-rule-update` to work. 

Is this a known bug? 

Thanks in advance.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment, marked as answer]
This is a known bug, thanks -- we're addressing it.

**Reply by DoughnutPeach** [reply]
Thanks. Looking forward to the update.
